cli/lib/kontena/scripts/completer

```diff
@@ -3,238 +3,7 @@
 
 # resolve bin path, ignoring symlinks
 require 'pathname'
-require 'yaml'
```

Comment:
It does come through some of the requires, but added it to completer.rb to be sure.